pkg/sql/subscriber.go

```diff
@@ -241,13 +254,7 @@ func (s *Subscriber) query(
 
 	offset, msg, err := s.config.SchemaAdapter.UnmarshalMessage(row)
 	if errors.Cause(err) == sql.ErrNoRows {
-		// wait until polling for the next message
-		logger.Debug("No more messages, waiting until next query", watermill.LogFields{
-			"wait_time": s.config.PollInterval,
-		})
-		tx.Rollback()
-		time.Sleep(s.config.PollInterval)
-		return "", nil
+		return NoMsgUUID, nil
```

Comment:
what do you think about using separate return parameter or custom err instead of `NoMsgUUID`? `NoMsgUUID` looks a bit magical for me :mage_man:

Reply:
SGTM